core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/data/impl/DataFeedServiceImpl.java

```diff
@@ -183,6 +182,19 @@ public void modifyDataFeed(Map<String, String> valuesMap) {
         }
     }
 
+    @Override
+    public void deleteDataFeed() {
```

Comment:
It is _desirable_ that methods that are called by user (this one is called by user via a servlet) *do report* their status to user. If I click a "Delete" button somewhere, do I actually delete stuff, or do I fail? If such a method fails, a user must get a notification. The most easy way to do this, is to make this method *throw* an exception. Then, catch the exception in the servlet. And either send back to the user the "OK" status, or some *error*. This is the RESTful approach